### PR TITLE
Remove issets from event_offline_receipt pay_later_text

### PR DESCRIPTION
**Pull request by eileenmcnaughton**
Overview
----------------------------------------
Remove issets from event_offline_receipt pay_later_text

Before
----------------------------------------

After
----------------------------------------

Technical Details
----------------------------------------

Comments
----------------------------------------

### CODE REVIEW

**Comment by eileenmcnaughton** [comment]
@colemanw @seamuslee001 looks like this one is ok...

File: xml/templates/message_templates/event_offline_receipt_html.tpl [hunk 1]
```diff
@@ -37,7 +37,7 @@
       <p>{ts}Once your registration has been reviewed, you will receive an email with a link to a web page where you can complete the registration process.{/ts}</p>
      {/if}
     {elseif !empty($is_pay_later)}
```
Comment (seamuslee001) [inline xml/templates/message_templates/event_offline_receipt_html.tpl:39]: @eileenmcnaughton is there a reason why your not getting rid of this empty as per L304?

Comment (eileenmcnaughton) [reply xml/templates/message_templates/event_offline_receipt_html.tpl:39]: @seamuslee001 I didn't spot it - if you want to merge this I'll include it in the next one due to the painfulness of the whole sql regen thing

Comment (seamuslee001) [reply xml/templates/message_templates/event_offline_receipt_html.tpl:39]: @eileenmcnaughton ok and there is a corresponding one in the txt at L28

**Pull request merged**